build/dependencies.props

```diff
@@ -14,6 +14,7 @@
     <MessagePackPackageVersion>1.9.11</MessagePackPackageVersion>
     <NewtonsoftJsonPackageVersion>11.0.2</NewtonsoftJsonPackageVersion>
     <SystemMemoryPackageVersion>4.5.3</SystemMemoryPackageVersion>
+    <MicrosoftAspNetCoreMvcWebApiCompatShimPackageVersion>2.2.0</MicrosoftAspNetCoreMvcWebApiCompatShimPackageVersion>
```

Comment:
This should not be in the list of dependencies. Why do we need it. It's deprecated in 3.0...

Reply:
I didn't notice that. Now it's just used in tests. I use `new HttpRequestMessageFeature().HttpRequestMessage` to get HttpRequestMessage in tests.

Reply:
Microsoft.AspNetCore.Mvc.WebApiCompatShim seems still be used in Azure Function, even though v3.0. https://github.com/Azure/azure-functions-host/blob/v3.0.13152/src/WebJobs.Script.WebHost/Controllers/HostController.cs#L14